src/Arcus.Messaging.Abstractions/MessageHandling/IMessageRouter.cs

```diff
@@ -10,24 +8,5 @@ namespace Arcus.Messaging.Abstractions.MessageHandling
     [Obsolete("Will be removed in v3.0 as only concrete implementations of message routing will be supported from now on")]
     public interface IMessageRouter
```

Comment:
Why don't we remove the interface entirely ?

Reply:
Step by step, keeping changes separated to reduce ripple effects.